dev/sbt-plugin/src/main/scala/com/lightbend/lagom/sbt/LagomPlugin.scala

```diff
@@ -228,6 +228,10 @@ object LagomReloadableService extends AutoPlugin {
   import autoImport._
 
   override def projectSettings = Seq(
+    compile in Compile := {
+      lagomVerifyProjectFamilyAlignment.value
+      (compile in Compile).value
+    },
```

Comment:
there are probably better ways of doing that, my limited sbt experience hit me today (again)